test/gtest/common/test_obj_size.cc

```diff
@@ -48,7 +48,7 @@ UCS_TEST_F(test_obj_size, size) {
 #else
     EXPECTED_SIZE(ucp_ep_t, 64);
     /* TODO reduce request size to 240 or less after removing old protocols state */
-    EXPECTED_SIZE(ucp_request_t, 296);
+    EXPECTED_SIZE(ucp_request_t, 312);
```

Comment:
no way to avoid this?

Reply:
I did two PRs to decrease UCP request size from 320 bytes to 296 bytes: #5829 and #5825 
I have the privilege to use at least 16 bytes :)
unfortunately, no way to reduce this - I tried to refactor UCP request `send` part, but it has `ucp_wireup_msg_t` in the union that could be reduced and it is our limit...